content/common/makefile/slides.mk

```diff
@@ -1,14 +1,22 @@
 RVMD = reveal-md
 
```

Comment:
I would use:
```
SLIDES ?= slides.mdpp
MEDIA_DIR ?= media
SITE ?= _site    # Is this configurable?
```

And I would use these variables throughout the `slides.mk` file. With the developer being able to define them in the including Makefile.

Reply:
The `_site` directory is configurable. I used it as a variable.